cmd/node-cache/main.go

```diff
@@ -83,6 +84,12 @@ func parseAndValidateFlags() (*app.ConfigParams, error) {
 		params.LocalIPs = append(params.LocalIPs, newIP)
 	}
 
+	// validate all the IPs have the same IP family
+	for _, ip := range params.LocalIPs {
+		if utilnet.IsIPv6(params.LocalIPs[0]) != utilnet.IsIPv6(ip) {
+			return params, fmt.Errorf("Invalid IP Family localip specified - %q", ip)
```

Comment:
Maybe reword to "Unexpected IP Family for localIP - %q, want IPv6=%v", ip, utilnet.IsIPv6(params.LocalIPs[0])

Reply:
yep, much better